core/src/net/sf/openrocket/rocketcomponent/Streamer.java

```diff
@@ -136,6 +138,14 @@ protected void loadFromPreset(ComponentPreset preset) {
 		if ( preset.has(ComponentPreset.WIDTH)) {
 			this.stripWidth = preset.get(ComponentPreset.WIDTH);
 		}
+		// ----	Set Cd value when preset is selected after manual Cd change
```

Comment:
I would leave out the `// ----` comment, so only `// Set Cd value when preset is selected after manual Cd change`

Reply:
Revised.